docker/ci/build.ps1

```diff
@@ -87,9 +85,51 @@ function Invoke-ContainerTest($Name, $Namespace = 'puppet', $Version = (Get-Cont
   Pop-Location
 }
 
-# removes any temporary containers / images used during builds
-function Clear-ContainerBuilds
+# removes temporary layers / containers / images used during builds
+# removes $Namespace/$Name images > 14 days old by default
+function Clear-ContainerBuilds(
+  $Namespace = 'puppet',
+  $Name,
+  $OlderThan = [DateTime]::Now.Subtract([TimeSpan]::FromDays(14)),
+  [Switch]
+  $Force = $false
```

Comment:
Note that in this PR I had to add support for `Force` to get rid of the images that have 2 tags.